OG_Format.adoc

```diff
@@ -194,7 +194,7 @@ ex.:
 |contributor_email |Email if the contributors to the glider mission. Multiple contributors’ emails are separated by commas. |PI email is mandatory |
 |contributor_id |Unique id of the contributors to the glider mission. Multiple contributors’ ids are separated by commas. |highly desirable |
 |contributor_role |Role of the contributors to the glider mission. Multiple contributors’ roles are separated by commas. |PI vocabulary is mandatory |
-|contributor_role_vocabulary |Controlled vocabulary for the roles used in the "contributors_role". Multiple contributors’ roles and vocabularies are separated by commas. |PI vocabulary is mandatory |https://orcid.org/[_https://orcid.org/_]
```

Comment:
Oh yeah, ORCID doesn't give role vocabulary. But shouldn't we use orcid for contributor_id?

Reply:
I think orcid can be a exemple of collection. They might be others. I remember that @justinbuck pointed to other collection of contributor's id.

Reply:
Hi @vturpin , try this http://vocab.nerc.ac.uk/search_nvs/W08/
Justin

Reply:
A note, we might need to expand the possible options on W08.